docs/tasks/README.md

```diff
@@ -3,6 +3,23 @@
 This section contains pages that show how to perform individual tasks.
 A task page shows how to do an operation with a short sequence of steps.
 
-## [Administer Cluster Resources](administer_cluster_resources.md)
+The tasks are designed to be followed by two personas: _batch administrators_
+and _batch users_.
 
-Learn how to setup Queues and ClusterQueues to administer quotas in a cluster.
\ No newline at end of file
+## As a batch administrator
+
+A _batch administrator_ manages the cluster infrastructure and establishes
+quotas and queues.
+
+### [Administer cluster quotas](administer_cluster_quotas.md)
+
+Learn how to setup Queues and ClusterQueues to administer quotas in a cluster.
+
```

Comment:
As a batch administator, learn how to [administer quotas in a cluster](administer_cluster_quotas.md) with `Queues` and `ClusterQueue`.